spec/features/users_creates_new_account_spec.rb

```diff
@@ -17,42 +40,72 @@
       ))
   end
 
-  scenario "views show page" do
-    user = create(:user)
+  scenario "successfully as admin" do
```

Comment:
This scenario lives in a file that is called `users_create_new_account_spec.rb` (really it should be singular: `user_creates_new_account_spec.rb`)

It should move to its own file.
Also, I would do different feature spec files for admin: for example: `admin_can_set_another_user_as_admin_spec.rb`.

Reply:
This spec file has too much in it.
```
feature "admin user accesses #index" do
  scenario "successfully" do
  end
end

feature "non-admin user accesses #index" do
  scenario "unsuccessfully" do
  end
end

feature "user creates new account" do
  scenario "successfully" do
  end

  scenario "successfully as admin" do
  end

  feature "views show page" do
    scenario "successfully" do
    end
  end

  feature "updates profile" do
    scenario "successfully" do
    end
  end

  feature "deletes account" do
    scenario "successfully" do
  end
end
```

Each one of these should be its own file:
- user creates an account
- user updates their details
- user delates their account
- admin views all users
- admin can set another user as admin

Reply:
Done in sha: b2bcac4